lib/stream-chat/client.rb

```diff
@@ -55,9 +58,13 @@ def initialize(api_key = '', api_secret = '', timeout = 6.0, **options)
     end
 
     # initializes a Stream Chat API Client from STREAM_KEY and STREAM_SECRET
-    # environmental variables.
+    # environmental variables. STREAM_CHAT_TIMEOUT and STREAM_CHAT_URL
+    # variables are optional.
     def self.from_env
```

Comment:
What do you think about something like this(not sure that the code is valid, but should explain the intent):
```ruby
   def self.from_env(**options)
      Client.new(ENV['STREAM_KEY'],
                 ENV['STREAM_SECRET'],
                 ENV['STREAM_CHAT_TIMEOUT'],
                 **({base_url: ENV['STREAM_CHAT_URL']}.merge(options)))
    end
```
?

Reply:
sure, why not